public/index.html

```diff
@@ -2,12 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <title>NearMe</title>
```

Comment:
Looks good! Think this might just need to be named in line with whatever we end up naming the app

Reply:
I think that's an older file! When you pull up this link and see the files - it looks like it has the correct title! let me know! https://github.com/the-collab-lab/tcl-20-whats-near-me/pull/4/files